src/app/file-explorer.js

```diff
@@ -160,12 +185,26 @@ function fileExplorer (appAPI, files) {
 
   function editModeOff (event) {
     var label = this
-    if (event.type === 'blur' || event.which === 27 || event.which === 13) {
+    if (event.which === 13) event.preventDefault()
+    if ((event.type === 'blur' || event.which === 27 || event.which === 13) && label.getAttribute('contenteditable')) {
+      var isFolder = label.getAttribute('isfolder') === 'true'
```

Comment:
aha, here it is being used. i see.
You could probably just use
`var isFolder = self.element.classname.indexOf(`folder`) !== -1`, because the node already contains the wanted information in it's classname.